File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {
        int n = nums.length;

        int res = 0;
        int count = 0;

        for(int i = 0; i < n; i++){
            if(count == 0){
                res = nums[i];
            }
            if(nums[i] == res) count++;
            else count--;
        }
        return res;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        hm = defaultdict(int)
        vis = set()
        st = set()
        for num in nums:
            st.add(num)
        oans = 0
        for num in nums:
            ans = 0
            cur = num
            while num in st and num not in vis:
                vis.add(num)
                num += 1
                ans += 1
            hm[cur] = max(ans+hm[num],hm[cur])
            oans = max(hm[cur],oans)
        return oans
        
        